test/test_resolution.py

```diff
@@ -135,6 +223,107 @@ def test_claim_without_kid(
     verify_receipt(claim, trust_store, receipt)
 
 
+class TestDIDMismatch:
+    """
+    Test DID resolution with an inconsistent DID document.
+
+    The "id" field of the document does not match the expected value.
+
+    This error scenario isn't particularly interesting in itself, but it is an
+    easy way to test the ledger's handling of errors during resolution.
+    """
+
+    @pytest.fixture
+    def identity(self, did_web) -> crypto.Signer:
+        issuer = did_web.generate_identifier()
+        private_key, _ = crypto.generate_keypair(kty="ec")
+        return crypto.Signer(private_key, issuer=issuer)
+
+    @pytest.fixture(autouse=True)
+    def create_document(self, did_web: DIDWebServer, identity: crypto.Signer):
```

Comment:
I find this instance of `autouse` a bit confusing when looking at the test functions. Maybe making it more explicit would make it easier to understand the tests?

Reply:
That's better, thanks.